app/controllers/search_controller.rb

```diff
@@ -3,7 +3,7 @@ class SearchController < ApplicationController
   before_action :search_summoner, only: [:search]
   
   def search
-    @name = params[:summoner]
+    @name = params[:summoner].downcase
     if @name.to_s.length<=1
```

Comment:
Not sure, but is the `.to_s` needed here?  I don't think you could call downcase if it wasn't already a string